reason-react-native/src/apis/Style.md

```diff
@@ -4,4 +4,1298 @@ title: Style
 officialDoc: https://facebook.github.io/react-native/docs/style
 ---
 
-@todo
+Styles accepted by most components (thanks to a `style` prop) are defined using
+style functions that prepare required object for React Native. The style names
+and values usually match how CSS works on the web, except names are written
+using camel casing, e.g `backgroundColor` rather than `background-color`.
+
+The `style` prop accept what the style functions return. That's the simplest way
```

Comment:
Perhaps better to write:
The `style` prop accepts the type returned by the style constructors. That's the simplest way

Reply:
Fixed